### PR TITLE
MAINT: Update Tutorials for multiline support

### LINKED ISSUE #38: Revise existing command blocks to use multiline formatting
Low-resolution displays make it hard to follow what is going on with long commands.

Blocked by #37

### PR DESCRIPTION
**Pull request by jakereps**
Resolves #38 

Pending merge of #43

### CODE REVIEW

**Comment by jairideout** [comment]
Looks like travis finished successfully

**Comment by thermokarst** [comment]
@jakereps: have you run `make html` on this yet?

**Comment by jakereps** [comment]
I partly ran it, my new environment didn't have every plugin, so it'd crash, but it got all the way to a `dada2` step. Was going to get a full distribution set up tomorrow to give it a run.

**Comment by jairideout** [comment]
Isn't it awesome how fast our tutorials run? 🐢 🐇

File: source/tutorials/moving-pictures.rst [hunk 1]
```diff
@@ -92,14 +103,22 @@ In these plots, the quality of the initial bases seems to be high, so we won't t
 
 .. command-block::
 
-   qiime dada2 denoise --i-demultiplexed-seqs demux.qza --p-trim-left 0 --p-trunc-len 100 --o-representative-sequences rep-seqs --o-table table
+   qiime dada2 denoise \
+     --i-demultiplexed-seqs demux.qza \
+     --p-trim-left 0 --p-trunc-len 100 \
```
Comment (jairideout) [inline source/tutorials/moving-pictures.rst]: Split into two lines

**Comment by jakereps** [comment]
Thanks @jairideout! Nice catch! Should be good to go.

**Comment by jairideout** [comment]
@jakereps I'm seeing an extra commit in here that introduces the changes in #43 to this PR diff. Can you fix that? Let me know if you have any issues.

**Comment by jakereps** [comment]
Oops, sorry about that @jairideout, I think I merged it in to run the test, since the lines were split but the directive functionality wasn't there. Fixed!

**Comment by jairideout** [comment]
Thanks @jakereps!

**Comment by ebolyen** [comment]
:tada: :tada: :+1: :100:
